inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/group/InitGroupCompleteListener.java

```diff
@@ -84,12 +81,9 @@ public ListenerResult listen(WorkflowContext context) throws WorkflowListenerExc
         updateGroupRequest.setVersion(existGroup.getVersion());
         groupService.update(updateGroupRequest, operator);
 
-        // update status of other related configs
-        streamService.updateStatus(groupId, null, StreamStatus.CONFIG_SUCCESSFUL.getCode(), operator);
-        if (InlongConstants.LIGHTWEIGHT_MODE.equals(groupInfo.getLightweight())) {
-            sourceService.updateStatus(groupId, null, SourceStatus.SOURCE_NORMAL.getCode(), operator);
-        } else {
-            sourceService.updateStatus(groupId, null, SourceStatus.TO_BE_ISSUED_ADD.getCode(), operator);
+        List<InlongStreamInfo> streamList = form.getStreamInfos();
+        for (InlongStreamInfo streamInfo : streamList) {
+            streamProcessOperation.startProcess(groupId, streamInfo.getInlongStreamId(), operator, false);
```

Comment:
Will the for loop take a long time? If yes, it is recommended to make it asynchronous.

Reply:
Yes,the `false` parameter is expressed in the method `streamProcessOperation.startProcess`
asynchronous